admin-part/src/main/kotlin/Methods.kt

```diff
@@ -90,6 +103,10 @@ internal fun List<Method>.diff(otherMethods: List<Method>): DiffMethods = if (an
     } else DiffMethods(new = this)
 } else DiffMethods(deleted = otherMethods)
 
+private fun MutableList<Method>.addMethod(value: Method) {
+    if ("lambda" !in value.name) add(value)
```

Comment:
Can we make "lambda" constant field ?
We also use it in admin-part/src/main/kotlin/jvm/Ext.kt file